Rakefile

```diff
@@ -289,21 +286,6 @@ namespace :publish do
 
 end
 
-namespace :distribute do
```

Comment:
These tasks don't work any more and rely on an outdated gem, so we removed them.